screens/wallet/SettingsScreen.tsx

```diff
@@ -131,7 +131,7 @@ const SettingsScreen = (props: ScreenProps<'SettingsScreen'>) => {
       onPress: () => biometricsAlert(),
     },
     {
-      title: 'Setup Staking Nodes',
+      title: 'Staking Nodes',
```

Comment:
Why are we not making Nodes lowercase too?

Reply:
yes